src/org/openstreetmap/josm/tools/PlatformHookWindows.java

```diff
@@ -486,7 +486,7 @@ protected void extendFontconfig(String templateFileName) {
                     if (prevValue != null && !prevValue.equals(value)) {
                         Logging.warn("extended font config - overriding ''{0}={1}'' with ''{2}''", key, prevValue, value);
                     }
-                    w.append(key + '=' + value + '\n');
+                    w.append(key).append(String.valueOf('=')).append(value).append(String.valueOf('\n'));
```

Comment:
Why are you using `String.valueOf`? `StringBuilder#append` has a `char` version.

Reply:
Good question, maybe I've ran into a situation when it's not possible to mix and used this way. Anyway good catch, thanks! Removed in [24ae86b](https://github.com/JOSM/josm/pull/124/commits/24ae86b6fd203192a765340c9bfd1fe9327b0255).